packages/react-tabs/src/components/Tab/renderTab.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { getSlots } from '@fluentui/react-utilities';
+import { tabShorthandProps } from './useTab';
+import type { TabState, TabSlots } from './Tab.types';
+
+/**
+ * Render the final JSX of Tab
+ */
+export const renderTab = (state: TabState) => {
+  const { slots, slotProps } = getSlots<TabSlots>(state, tabShorthandProps);
+
+  return (
+    <slots.root role="tab" {...slotProps.root}>
```

Comment:
why is the role explicitly here (it is also in useTab, which is what other components hae)

Reply:
Good catch.  I must have put it there early on before I put it on useTab.